website/content/docs/other-specifications/acl-policy.mdx

```diff
@@ -149,6 +149,11 @@ of `capabilities`. This includes:
   and listing external volumes and snapshots.
 - `csi-mount-volume` - Allows jobs to be submitted that claim a CSI volume. This
   implicitly grants `csi-read-volume`.
+- `host-volume-create` - Allows creating a dynamic host volume. This implicitly grants `host-volume-read`.
+- `host-volume-delete` - Allows deleting a dynamic host volume.
+- `host-volume-read` - Allows inspecting dynamic host volumes.
+- `host-volume-register` - Allows registering dynamic host volumes that have been created without a plugin. This implicitly grants `host-volume-read` and `host-volume-create`.
+- `host-volume-write` - Allows all write operations on dynamic host volumes. This implicitly grants `host-volume-read`, `host-volume-create`, `host-volume-register`, and `host-volume-delete`.
```

Comment:
Is there a reason `create` and `delete` refer to a singular volume, and the remaining ACLs to plural volumes? </nitpick>

Reply:
Good catch. Done.